docs/source/en/index.mdx

```diff
@@ -12,18 +12,13 @@ specific language governing permissions and limitations under the License.
 
 # 🤗 Transformers
 
-State-of-the-art Machine Learning for PyTorch, TensorFlow and JAX.
+State-of-the-art Transformers models in PyTorch, TensorFlow, and JAX for research and production.
```

Comment:
I wouldn't change the subtitle, which is the same as the README.

Reply:
Same comment here for me.

Reply:
What if I also changed the subtitle in the README? 😏

I'm ok if we don't change the subtitle though, I just feel like this is a slightly more precise description of what `transformers` is. It describes exactly what the library offers and for what purpose

Reply:
The library has multiple models that are not Transformers based, so this new suggestion is factually wrong, not more precise ;-)